stl/inc/span

```diff
@@ -523,15 +532,15 @@ public:
             _STL_VERIFY(_Count <= this->size(), "Count out of range in span::first()");
         }
 #endif // _CONTAINER_DEBUG_LEVEL > 0
-        return span<element_type, _Count>{_Mydata, _Count};
+        return span<element_type, _Count>{this->_Mydata, _Count};

```

Comment:
I don't mind a `this->` or two, but these are contributing too much noise. Could we please pull in `_Mydata` and `size` with using declarations and eliminate all of the `this->` occurrences?

Reply:
I did it for _Mydata but I belive I cannot do that for a member function?!

Reply:
> I did it for _Mydata but I beleive I cannot do that for a member function?!

`using _Span_extent_type<_Ty, _Extent>::size;` is the magic. (Change pushed.)

Reply:
Is it obvious that `_Mybase::_Mydata` works and `_Mybase::size` not?

Reply:
Nitpick: You could have put it to the other using declarations